nifi-nar-bundles/nifi-riemann-bundle/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>nifi-nar-bundles</artifactId>
+    <groupId>org.apache.nifi</groupId>
+    <version>0.3.1-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>nifi-riemann-bundle</artifactId>
+  <packaging>pom</packaging>
+  <modules>
+    <module>nifi-riemann-processors</module>
+    <module>nifi-riemann-nar</module>
+  </modules>
+  <repositories>
+    <repository>
+      <id>clojars.org</id>
+      <url>http://clojars.org/repo</url>
+    </repository>
+  </repositories>
+  <dependencies>
+    <dependency>
+      <groupId>com.aphyr</groupId>
+      <artifactId>riemann-java-client</artifactId>
```

Comment:
one eas(ier) way to check the transitive dependencies is to use "mvn dependency:list" within your module. that will give you a set of group/artifact/versions you can use to grab individual licenses.

since we build the modules with the resource plugin, you can just go to the target/ directory and unpack oyur jar file. within, there will be a file `META-INF/DEPENDENCIES` that will list what all the transitive dependency poms claim the license is. Usually they'll include the library's website if you want to verify.